metadata/integrations.yml

```
@@ -109,4 +109,19 @@ live_agents:
+    required_fields:
+      - restaurant_name
+      - branch_name
+      - restaurant_id
+      - sync_type:
+          - push_menu
+          - item_toggle
+    optional_fields:
+      - access_token
+      - catalog_id
+    disabled_fields:
+      - provider
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)